### ISSUE TITLE
Accessibility: Cannot select pages for batch actions

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-cms#3114]
Related: https://github.com/silverstripe/silverstripe-cms/issues/3101

When batch actions is activated, it's impossible to select a page. This means you can't perform batch actions using the keyboard.

<img width="1140" height="974" alt="Image" src="https://github.com/user-attachments/assets/9a129b73-a589-47e0-a152-eeddd7472ea7" />

## Notes

- https://www.w3.org/WAI/ARIA/apg/patterns/treeview/ provides recommended keybinds (see Selection in multi-select trees) and appropriate attributes to add (see WAI-ARIA Roles, States, and Properties)

## Reproduction steps

1. Go to `/admin/pages`
3. Go to the Page tree
5. Tab to the batch action button and activate it
9. Tab through a page that needs to be published
11. Press the space bar to select that page (it doesn't get selected)

**Comment by emteknetnz** [comment]
Will get fixed in this issue https://github.com/silverstripe/silverstripe-cms/issues/3135
